test/Microsoft.AspNet.Mvc.ModelBinding.Test/Metadata/ModelMetadataProviderTest.cs

```diff
@@ -621,14 +665,6 @@ public DataTypeWithCustomEditFormat() : base("Custom datatype")
             }
         }
 
-        private void ActionWithoutBindAttribute(User param)
-        {
-        }
-
-        private void ActionWithBindAttribute([Bind(new string[] { "IsAdmin" }, Prefix = "ParameterPrefix")] User param)
-        {
-        }
-
```

Comment:
unused, this should have been removed as part of removing `GetMetadataForParameter`